packages/orchestration/test/examples/basic-flows.contract.test.ts

```diff
@@ -211,7 +211,7 @@ test('Deposit, Withdraw errors - LocalOrchAccount', async t => {
     });
     await t.throwsAsync(vt.when(E(withdrawSeat).getOfferResult()), {
       message:
-        'One or more withdrawals failed ["[RangeError: -10 is negative]"]',
+        'One or more withdrawals failed ["[Error: cannot grab 10uist coins: spendable balance 0uist is smaller than 10uist: insufficient funds]"]',
```

Comment:
There's an upcoming incompatibility with v0.47 I'd like to avoid.  Zero coins (`0uist` here) is rendered as an empty string in the future, so:

```suggestion
        /^One or more withdrawals failed \["\[Error: cannot grab 10uist coins: spendable balance (0uist)? is smaller than 10uist: insufficient funds\]"\]$/,
```